utils/travis/steps/install.sh

```diff
@@ -8,12 +8,16 @@ export PLAY_TEST=true
 export MP_TEST=true
 export WML_TEST_TIME=15
 export BOOST_TEST=true
+export LTO=false
 
 if [ "$OPT" = "-O0" ]; then
     export EXTRA_FLAGS_RELEASE="-O0"
     export PLAY_TEST=false
     export MP_TEST=false
     export WML_TEST_TIME=20
+else
+# change to true to enable LTO on optimized, non-xcode builds
```

Comment:
Please indent this line as much as the following line.

Reply:
Done.